make.jl

```diff
@@ -11,7 +11,8 @@ makedocs(;
     ),
     pages = [
         "Home" => "index.md",
-        "Ecosystem" => "ecosystem.md"
+        "Ecosystem" => "ecosystem.md",
+        "Tutorials" => "tutorials.md"
```

Comment:
Suggestion
```
        "Tutorials" => [ ], #...
```

Reply:
I tried moving tutorials.md into tutorials/index.md and adding the tutorial to the sidebar. 
Was this what you had in mind?

![image](https://user-images.githubusercontent.com/7330605/181536234-0b7c4dea-9d1b-495b-a1a4-2fa7b4750b95.png)